src/cascadia/TerminalSettingsModel/KeyChordSerialization.cpp

```diff
@@ -124,7 +124,7 @@ static int32_t parseNumericCode(const std::wstring_view& str, const std::wstring
 static KeyChord _fromString(std::wstring_view wstr)
 {
     using nameToVkeyPair = std::pair<std::wstring_view, int32_t>;
-    static const til::static_map nameToVkey{
+    static constinit til::static_map nameToVkey{

```

Comment:
This is the primary benefit of the PR. It's not really important, I just did it because I noticed it and wanted to show `til::static_map` around.

Reply:
THANK YOUTHANK YOUTHANK YOU